cli/src/config-schema.json

```diff
@@ -108,6 +108,35 @@
                     "description": "Pager to use for displaying command output",
                     "default": "less -FRX"
                 },
+                "builtin-pager": {
+                    "type": "object",
+                    "description": "':builtin' pager configuration",
+                    "properties": {
+                        "alternate-screen": {
```

Comment:
nit: it sounds like "(use) alternate screen" (noun). How about `reset-screen` or `clear-screen`?

Oh, but `never` means "no init" + "quit automatically". I feel "interface-mode" is better in that case.

Reply:
I changed it to `clear-screen`.

If we went with `interface-mode`, I'm not sure what the options should be so that they are not too long and clear. I don't like the choices Streampager made for these names.

But I'm unsure what's best here.